packages/unenv-preset/package.json

```diff
@@ -54,7 +54,7 @@
 	},
 	"peerDependencies": {
 		"unenv": "2.0.0-rc.17",
```

Comment:
We'll need to update the vite-plugin to depend upon this version too I think

Reply:
https://github.com/cloudflare/workers-sdk/pull/9228